t/t0022-jj-reader.t

```diff
@@ -8,23 +8,25 @@ jj=${FLUX_BUILD_DIR}/t/sched-simple/jj-reader
 y2j="flux python ${SHARNESS_TEST_SRCDIR}/jobspec/y2j.py"
 
 test_expect_success HAVE_JQ 'jj-reader: unexpected version throws error' '
-	flux jobspec srun hostname | jq ".version = 2" >input.$test_count &&
```

Comment:
In commit message: missing word(?) "the flux jobspec _command_ is being removed"?